lib/iris/tests/unit/fileformats/grib/test_GribWrapper.py

```diff
@@ -30,9 +30,10 @@
 
 from iris._lazy_data import as_concrete_data, is_lazy_data
 from iris.exceptions import TranslationError
-from iris.fileformats.grib import GribWrapper, GribDataProxy
+import iris.fileformats.grib as grib
+from iris.fileformats.grib import GribWrapper
```

Comment:
Yes, I was unsure about that.  You don't have to apologize, I am good at taking criticism.